Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS builder
```

Comment:
The main branch is not merged into actual version branches. The downstream version branches are in sync with upstream with minimal or no carry overs. Also, these repositories are going to be managed by the build pipeline?

Reply:
> The main branch is not merged into actual version branches.

oh ok, I will create PR against specific release branch for now 1.15, and later while creating build configs for older releases, I will cherrypick the change to required releases branches. Please let me know WDYT.

> The downstream version branches are in sync with upstream with minimal or no carry overs

Yeah, my understanding is based on how openshift specific Dockerfiles are added in other upstream fork repositories. If carry commit is not desired. Please let me know how I could manage this. Since it's specific for openshift, I think maintainers won't allow it be added in upstream.

> Also, these repositories are going to be managed by the build pipeline?

Sorry I don't understand the question, pipelines added by konflux-bot are for creating build images only.

Reply:
>Sorry I don't understand the question, pipelines added by konflux-bot are for creating build images only.

Is there an automation to automatically cherry-pick from main branch for the release branch?

Reply:
> Yeah, my understanding is based on how openshift specific Dockerfiles are added in other upstream fork repositories. If carry commit is not desired. Please let me know how I could manage this. Since it's specific for openshift, I think maintainers won't allow it be added in upstream.

Following might be the options to consider, add a separate openshift folder and maintain openshift specific files so that there are no conflicts during rebase. Or, maintain the build files as part of cert manager operator and have pipeline use the path during the build. Whichever is easier to maintain can be opted. In case of this repository, the cherry pick should be done and maintained manually on the release branches, whereas, cert-manager operator will have the configuration during the branch-cut.

Reply:
> Is there an automation to automatically cherry-pick from main branch for the release branch?

I am not aware of it, but `/cherry-pick` can be used for creating a PR.

Also, created https://github.com/openshift/jetstack-cert-manager/pull/23 for release-1.15. PTAL.